biochatter/llm_connect.py

```diff
@@ -257,18 +260,26 @@ def _inject_context(self, text: str):
                 for agent in self.rag_agents:
                     if not agent.use_prompt:
                         continue
-                    statements = statements + [
-                        doc[0] for doc in agent.generate_responses(text)
-                    ]
+                    try:
+                        docs = agent.generate_responses(text)
+                        statements = statements + [
+                            doc[0] for doc in agent.generate_responses(text)
```

Comment:
shouldn't this be `for doc in docs`, as below?

Reply:
Good catch.